runtime/src/accounts.rs

```diff
@@ -467,6 +469,87 @@ impl Accounts {
             .collect()
     }
 
+    pub fn load_accounts_parallel<'a>(
```

Comment:
Hmmm, this is not a bad idea, but I'm wondering if we can get even better perf if we scheduled the entire load + execute in pathway in parallel. Might be worth exploring 😃

Reply:
Hmm, the reason why I think that's not necessarily a good idea is that the execute portion is CPU-bound.

The purpose of making the load portion parallel or async was to reduce the tail-end critical path of sequentially loading 128 TXs with up to 20-256 accounts each.

In general tx parallelism can already be handled by spawning more banking or replay threads.

Reply:
Example API:
`get_accounts_async_batch(keys: &[Pubkey], to_load: &[Res<()>]) -> Vec<LoadedAccount>`